server/chat-plugins/othermetas.ts

```diff
@@ -210,10 +214,12 @@ export const commands: ChatCommands = {
 		const deltas: StoneDeltas = {
 			baseStats: Object.create(null),
 			weighthg: megaSpecies.weighthg - baseSpecies.weighthg,
+			bst: 0,
```

Comment:
```suggestion
			bst: megaSpecies.bst - baseSpecies.bst,
```
(Along with the removal of line 222 of course.)

Reply:
I'm unsure why I didnt think of this in the first place lol